Aztec.xcodeproj/project.pbxproj

```diff
@@ -967,8 +967,7 @@
 				F1DC21B22199D58B007C189E /* ElementToAttributedString */,
 				F1E75630215B0C9E004BC254 /* StringAttributesToAttributes */,
 			);
-			name = Converters;
-			path = "New Group";
+			path = Converters;
```

Comment:
I noticed that this group was not properly renamed on disk and renamed it so it shows up correctly in SPM.